pkg/exchange/max/stream.go

```diff
@@ -255,13 +255,26 @@ func (s *Stream) handleBookEvent(ex *Exchange) func(e max.BookEvent) {
 			return
 		}
 
-		if err := f.AddUpdate(types.SliceOrderBook{
-			Symbol: symbol,
-			Time:   e.Time(),
-			Bids:   e.Bids,
-			Asks:   e.Asks,
-		}, e.FirstUpdateID, e.LastUpdateID); err != nil {
-			log.WithError(err).Errorf("found missing %s update event", e.Market)
+		if e.Event == "snapshot" {
```

Comment:
do we have a constant for this? in the maxapi/v3 package.

Reply:
No, we bypass this to book event.